Refresh.GameServer/Types/Levels/CustomRewards/CustomReward.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Xml.Serialization;
+using Realms;
+
+namespace Refresh.GameServer.Types.Levels.CustomRewards;
+
+[XmlType("customReward")]
+public partial class CustomReward : IEmbeddedObject
+{
+    [XmlAttribute(AttributeName = "id")] public int Id { get; set; }
+    [XmlElement("enabled")] public bool Enabled { get; set; }
+    [XmlElement("description")] public string? Description { get; set; }
```

Comment:
Description or title? In the video you posted on Discord, it seemed like more of a title to me. I think it's acceptable to rename as a lot of things in Refresh fix naming (GameLevel inplace of GameSlot for instance)